### ISSUE TITLE
Error when trying to use just the memory feature

### ISSUE THREAD

**Issue by jasonpkovalski** [TerminalWitchcraft/actix-ratelimit#14]
**Reproduce**: Use the following in `Cargo.toml`: `actix-ratelimit = { version = "0.3.0", default-features = false, features = ["memory"] }`

`cargo check` output:

```
error[E0432]: unresolved import `stores::memory`
   --> /Users/jasonpkovalski/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-ratelimit-0.3.0/src/lib.rs:194:17
    |
194 | pub use stores::memory::{MemoryStore, MemoryStoreActor};
    |                 ^^^^^^ could not find `memory` in `stores`
```

**Comment by TerminalWitchcraft** [comment]
Thanks for reporting, I’ll look into it asap

**Comment by TerminalWitchcraft** [comment]
Fixed in version 0.3.1
